upstream-community-operators/aws-service/aws-service.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,194 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: aws-service-operator.v0.0.1
+  namespace: placeholder
+  annotations:
+    categories: "Cloud Provider"
+    description: The AWS Service Operator allows you to manage AWS resources using Kubernetes Custom Resource Definitions.
+    containerImage: awsserviceoperator/aws-service-operator:v0.0.1-alpha3
+    createdAt: 2019-02-09T12:00:00Z
+    support: AWS Labs
+spec:
+  displayName: AWS Service Operator
+  description: >
+    The AWS Service Operator allows you to manage AWS resources using Kubernetes
+    Custom Resource Definitions. Using the AWS Service Operator enables a gitops
+    workflow to drive your infrastructure to the desired state leveraging
+    Kubernetes Custom Resource Definitions (CRD), the Kubernetes internal
+    control loop, and AWS CloudFormation orchestration.
+
+
+    In order to run the Operator the node will need AWS IAM privileges. This can
+    be done through a service like kube2iam. If you are using an Amazon Elastic
```

Comment:
Given these support `markdown` should we wrap tools like `kube2iam` in backticks?

Reply:
I like the idea of drawing attention to kube2iam, I'll add a hyperlink to the project's github repository.